contentful_management/resource.py

```diff
@@ -298,11 +298,17 @@ def __setattr__(self, name, value):
         if name not in ['raw', 'sys', 'default_locale',
                         '_client', '_fields']:
             locale = self._locale()
-            if name in self._fields.get(locale, {}):
+            if (name in self._fields.get(locale, {}) or
+                    self._is_missing_field(name)):
+                if locale not in self._fields:
+                    self._fields[locale] = {}
                 self._fields[locale][name] = value
                 return self._fields[locale][name]
         return super(FieldsResource, self).__setattr__(name, value)
 
+    def _is_missing_field(self, name):
```

Comment:
basically you are returning `False` here because this call should be handled by a subclass and is therefore like an abstract method right? If the answer is yes, I think it would be nice to have a comment saying that subclasses should override this method.